src/pages/references/node-configuration.md

```diff
@@ -0,0 +1,489 @@
+---
+title: Node Configuration
+description: Configuration parameters and options for the stacks-node binary
+icon: TestnetIcon
+images:
+  large: /images/pages/testnet.svg
+  sm: /images/pages/testnet-sm.svg
+---
+
+## Table of Contents
```

Comment:
we actually preview the contents on the right-side page nav. I'd remove the ToC from the markdown

Reply:
Is there any way to customize the preview on the right side? It only lists the top-level `##` headings, and if I can get it to resemble my ToC that would be perfect.

Reply:
unfortunately, we don't have an option to set this per page. the two-level heading is critical for longer pages and seemed a good balance overall across the docs. I wouldn't worry about this too much. I'd assume that devs going to the sources would control+f anyways